data/pgroonga.sql

```diff
@@ -435,9 +435,18 @@ CREATE FUNCTION pgroonga_wal_set_applied_position()
 	STRICT;
 
 CREATE FUNCTION pgroonga_list_lagged_indexes()
-	RETURNS void /* todo */
-	AS 'MODULE_PATHNAME', 'pgroonga_list_lagged_indexes'
-	LANGUAGE C
+	RETURNS TABLE (name text) AS $$
+		SELECT name FROM pgroonga_wal_status()
+		WHERE current_block != last_block
+			OR current_offset != current_offset
+			OR current_size != last_size
+		UNION
+		SELECT name FROM pgroonga_wal_status()
+		WHERE EXISTS(
```

Comment:
Can we use one `SELECT`?

```suggestion
		OR EXISTS(
```

Reply:
Good point.